server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -178,8 +178,9 @@ static TransportVersion def(int id) {
     public static final TransportVersion TIMEOUT_GET_PARAM_FOR_RESOLVE_CLUSTER = def(8_838_0_00);
     public static final TransportVersion INFERENCE_REQUEST_ADAPTIVE_RATE_LIMITING = def(8_839_0_00);
     public static final TransportVersion ML_INFERENCE_IBM_WATSONX_RERANK_ADDED = def(8_840_0_00);
-    public static final TransportVersion COHERE_BIT_EMBEDDING_TYPE_SUPPORT_ADDED_BACKPORT_8_X = def(8_840_0_01);
-    public static final TransportVersion REMOVE_ALL_APPLICABLE_SELECTOR_BACKPORT_8_X = def(8_840_0_02);
+    public static final TransportVersion ELASTICSEARCH_8_19 = def(8_841_0_00);
```

Comment:
nit: in the automation PR for creating transport versions for each release, we've settled on `INITIAL_ELASTICSEARCH_M_NN`, let's match that here.